apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -157,6 +153,26 @@ Response cachedHttpResponse(String cacheKey) throws IOException {
     }
   }
 
+  private <D extends Operation.Data, T, V extends Operation.Variables> RealApolloCall<T> newCall(
+      @Nonnull Operation<D, T, V> operation) {
+    ResponseFieldMapper responseFieldMapper = responseFieldMapper(operation);
+    return new RealApolloCall<T>(operation, serverUrl, httpCallFactory, httpCache, defaultHttpCachePolicy, moshi,
+        responseFieldMapper, customTypeAdapters, apolloStore, defaultCacheControl, defaultCacheHeaders,
+        dispatcher, logger, applicationInterceptors, Collections.<OperationName>emptyList());
+  }
+
+  private ResponseFieldMapper responseFieldMapper(Operation operation) {
+    ResponseFieldMapper responseFieldMapper;
+    synchronized (responseFieldMapperPool) {
+      responseFieldMapper = responseFieldMapperPool.get(operation.getClass());
+      if (responseFieldMapper == null) {
```

Comment:
Sorry if this is out of scope for this pr but is there a way way can do a null object pattern or some other way to show absence without using `null`.  also missing nullability annotations